src/tests/functional/plugin/shared/include/behavior/ov_plugin/core_threading.hpp

```diff
@@ -411,6 +413,7 @@ class CoreThreadingTestsWithIter : public testing::WithParamInterface<CoreThread
         std::tie(target_device, config) = std::get<0>(GetParam());
         numThreads = std::get<1>(GetParam());
         numIterations = std::get<2>(GetParam());
+        APIBaseTest::SetUp();
```

Comment:
not need here

Reply:
Why ? If APIBaseTest::SetUp is not in this SetUp, tests will reported as fail.

Reply:
Correct. Sorry. I thought that it is `getTestCaseName` :)